cluster-scope/overlays/nerc-ocp-obs/kustomization.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  nerc.mghpcc.org/kustomized: "true"
+
+resources:
+- ../common
+- ../../bundles/node-feature-discovery
+- ../../bundles/patch-operator
```

Comment:
Do we need the patch operator? We've tried to avoid it as much as possible.

Reply:
you are right. We should not need it.

Reply:
you are right. We should not need it.